test/jdk/java/awt/datatransfer/Independence/IndependenceAWTTest.java

```diff
@@ -65,46 +88,44 @@ public void focusGained(FocusEvent fe) {
         tf2 = new TextField();
```

Comment:
Line 82

```patch
--- a/test/jdk/java/awt/datatransfer/Independence/IndependenceAWTTest.java
+++ b/test/jdk/java/awt/datatransfer/Independence/IndependenceAWTTest.java
@@ -80,7 +80,7 @@ private static void createAndShowUI() {
         tf1 = new TextField();
         tf1.addFocusListener(new FocusAdapter() {
             public void focusGained(FocusEvent fe) {
-                tf1.setText("Clipboards_Independance_Testing");
+                tf1.setText("Clipboards_Independence_Testing");
             }
         });
```

Same for the `IndependenceSwingTest`

Reply:
The typos are still there.

Reply:
Missed it earlier. Fixed.